### DISCUSSION TITLE
Routing protocols for mobile ad hoc networks - Showcase

### DISCUSSION THREAD

**Discussion by adamgeorge309** [inet-framework/inet#602] [Showcases]
Discuss the "Routing protocols for mobile ad hoc networks" showcase here. Ideas and recommendations are welcome.

**Comment by esfandia** [comment]
How does this showcase, and the modules it uses, relate to [inetmanet](https://github.com/aarizaq/inetmanet-3.x), an older package that seems to have been part of earlier versions of inet? Is everything going to be merged eventually? What are the plans for supporting OLSR? Thanks in advance for your help and guidance.

**Comment by rhornig** [comment]
INETMANET contains additional MANET routing protocols compared to INET. Those protocols are third party code that were modified a bit and the INETMANE project created an interface that allows the INET infrastructure to work with those. As these are parts of other project they will not be merged into INET, but remain in INETMANET. If you need those, you should use INETMANET. We are not aware of anyone who have created a native omnet implementation for OLSR.
